src/interactions/PathTween.swift

```diff
@@ -69,6 +69,14 @@ public final class PathTween: Interaction, Togglable, Stateful {
   }
 
   /**
+   Initializes a path tween instance with its required properties.
+   */
+  convenience public init(duration: DispatchTimeInterval, path: CGPath, system: @escaping PathTweenToStream<CGPoint> = coreAnimation, timeline: Timeline? = nil) {
+    let durationCGFloat = duration.toSeconds()
```

Comment:
One last small change and then we're good to merge: Rename these variables throughout to `let durationInSeconds` so that we drop the CGFloat from the variable names.